src/thor/triplegbuilder.cc

```diff
@@ -1151,11 +1151,15 @@ TripLeg_Edge* AddTripEdge(const AttributesController& controller,
     trip_edge->set_speed(speed);
   }
 
-  uint8_t kAccess = 0;
+  uint16_t kAccess = 0;
```

Comment:
Widened because this hasn't accommodated the access mask in some time but we just only used the lower values previously.